common/src/db/query.rs

```diff
@@ -403,9 +400,24 @@ impl FromStr for Operator {
 // Internal helpers
 /////////////////////////////////////////////////////////////////////////
 
+#[derive(Debug)]
+enum ValueOrSimpleExpr {
```

Comment:
I don't love the name, but it's private, so whatevs. I'd prefer something shorter, e.g. `V`, `Rhs`, `Expr`. Maybe `Expr::Value` and `Expr::Simple`? I dunno. If you like it as it is, I'm good.

Reply:
I'm preferring verbosity these days personally. 'Expr' I'd find ambiguous with SeaORM's own Expr, 'V' feels terse, 'Rhs' maybe, but... eh?